apps/notifications/src/interfaces/alert-events/handlers/chain-events/chain-events.handler.ts

```diff
@@ -20,7 +22,27 @@ export class ChainEventsHandler {
     dto: ChainBlockCreatedDto
   })
   async processBlock(block: ChainBlockCreatedDto) {
-    await this.deploymentBalanceAlertsService.alertFor(block, message => this.brokerService.publish(eventKeyRegistry.createNotification, message));
+    const results = await Promise.allSettled([
+      this.deploymentBalanceAlertsService.alertFor(block, message => this.brokerService.publish(eventKeyRegistry.createNotification, message)),
+      this.walletBalanceAlertsService.alertFor(block, async message => this.brokerService.publish(eventKeyRegistry.createNotification, message))
```

Comment:
question: Does that mean we fetch the balance from the api every block for every alerts?